algorithms/C/README.md

```diff
@@ -21,6 +21,9 @@
 - [Reverse a Linked List](linked-lists/reverse-linkedlists.c)
 - [Doubly Linked List](linked-lists/doubly-linked-list.c)
 
+## Maths
+- [Palindrome Number](Maths/palindrome.c)
```

Comment:
Wich is the difference between palindrome on number and string?

Reply:
There is no such difference between palindrome number and string in theory,
But this program only suits for number(s) that's why I keep it in maths folder and will later add some other mathematical programs like Armstrong, prime, even-odd, Fibonacci etc in this math folder.